etc/lime-elements.api.md

```diff
@@ -755,8 +755,101 @@ export enum FormLayoutType {
 }
 
 // @public
-export interface FormSchema extends JSONSchema7 {
-}
+export interface FormSchema<T extends Record<string, any> = any> extends JSONSchema7 {
+    // Warning: (ae-unresolved-inheritdoc-base) The @inheritDoc tag needs a TSDoc declaration reference; signature matching is not supported yet

```

Comment:
There seems to be a lot of warnings generated here :thinking:

Reply:
It looks like this is due to using inheritDoc on properties that doesn't have docs.
Maybe that can be fixed, by looking what properties that have docstrings. Either we add our own OR we just remove the inheritdoc tag.

As I cannot run the api-extractor locally maybe you can do this for me @jgroth?

Reply:
I have fixed this in a fixup commit 👍 
But as before I cannot run the api-update so you have to do it for me 👎

Reply:
Yeah I can run it :smile: 
Perhaps we should move the logics of this `.sh` file to a `.js` file instead :smile:

Reply:
If you rebase on latest `main`, the `.sh` file is no more 😉

Reply:
Trying it out as we speak 👍

Reply:
It looks like it runs perfectly for me now ❤️